miro_preprocessor/Makefile

```diff
@@ -0,0 +1,53 @@
+ROOT = $(shell git rev-parse --show-toplevel)
+
+ifneq ($(ROOT), $(shell pwd))
+	include $(ROOT)/shared.Makefile
+endif
+
+
+xml_to_json_converter-build: xml_to_json_converter-test $(ROOT)/.docker/image_builder
+	$(ROOT)/scripts/run_docker_in_docker.sh \
+		-v $(ROOT):/repo \
+		image_builder:latest \
+		--project=xml_to_json_converter \
+		--file=/repo/miro_preprocessor/xml_to_json_converter.Dockerfile
+
+xml_to_json_converter-test: $(ROOT)/.docker/python3.6_ci
+	docker run \
+		-v $(ROOT)/miro_preprocessor/xml_to_json_converter:/data \
+		-e OP=test -e FIND_MATCH_PATHS='/data' \
+		python3.6_ci:latest
+
+xml_to_json_converter-publish: xml_to_json_converter-build $(ROOT)/.docker/publish_service_to_aws
+	$(ROOT)/scripts/run_docker_with_aws_credentials.sh \
+	    -v /var/run/docker.sock:/var/run/docker.sock \
+		-v $(ROOT):/repo \
```

Comment:
Here we need docker in docker & AWS credentials - maybe we should thinking about changing this helper scripts so it's possible to compose requirements.

Reply:
For sure. I have some ideas about how we’d do this, but that’ll be a separate patch.

Reply:
TODO: Open an issue to track fixing this